### PR TITLE
Update dividers.txt

### PR DESCRIPTION
**Pull request by N8python**
## Description
Fix issue, make all divs 12 chars.
## Context
This pull request fixes: [https://github.com/area4lib/area4/issues/157]the issue here

### CODE REVIEW

File: area4/dividers.txt [hunk 1]
```diff
@@ -33,7 +33,7 @@ r(`,,,`)r r(`,,,`)r r(`,,,`)r
 <><><><><><>
 ^,^,^,^,^,^,
 &*&*&*&*&*&*
-Random number divider
```
Comment (RDIL) [inline area4/dividers.txt]: This is ignored, please revert

File: area4/dividers.txt [hunk 2]
```diff
@@ -12,14 +12,14 @@ ____________
 ////////////
 ||||||||||||
 ~~~~~~~~~~~~
-\\\\\\\\\\\\\\\\\\\\\\\\
+\\\\\\\\\\\\
 ☕☕☕☕☕☕☕☕☕☕☕☕
 ++++++++++++
-r(`,,,`)r r(`,,,`)r r(`,,,`)r 
-( ͡° ͜ʖ ͡°)( ͡° ͜ʖ ͡°)( ͡° ͜ʖ ͡°)( ͡° ͜ʖ ͡°)
+r(`,,,`)r r(
```
Comment (RDIL) [inline area4/dividers.txt]:
```suggestion
r(`,,,`)r
```

File: area4/dividers.txt [hunk 3]
```diff
@@ -12,14 +12,14 @@ ____________
 ////////////
 ||||||||||||
 ~~~~~~~~~~~~
-\\\\\\\\\\\\\\\\\\\\\\\\
+\\\\\\\\\\\\
```
Comment (RDIL) [inline area4/dividers.txt]:
```suggestion
\\\\\\\\\\\\\\\\\\\\\\\\
```
these are escaped

File: area4/dividers.txt [hunk 4]
```diff
@@ -12,14 +12,14 @@ ____________
...
 ☕☕☕☕☕☕☕☕☕☕☕☕
 ++++++++++++
-r(`,,,`)r r(`,,,`)r r(`,,,`)r 
-( ͡° ͜ʖ ͡°)( ͡° ͜ʖ ͡°)( ͡° ͜ʖ ͡°)( ͡° ͜ʖ ͡°)
+r(`,,,`)r r(
+( ͡° ͜ʖ ͡°)(
```
Comment (RDIL) [inline area4/dividers.txt]:
```suggestion
( ͡° ͜ʖ ͡°)
```

File: area4/dividers.txt [hunk 5]
```diff
@@ -12,14 +12,14 @@ ____________
...
 &&&&&&&&&&&&
 ^^^^^^^^^^^^
-¯\\_(ツ)_/¯ ¯\\_(ツ)_/¯ ¯\\_(ツ)_/¯ ¯\\_(ツ)_/¯
+¯\\_(ツ)_/¯ ¯
```
Comment (RDIL) [inline area4/dividers.txt]:
```suggestion
¯\\_(ツ)_/¯
```

File: area4/dividers.txt (hunk 1 above)
Comment (N8python) [reply area4/dividers.txt]: You reverted. Sorry. I just wrote a program to reduce each line to 12 chars.

**Comment by N8python** [comment]
Awesome!

**Pull request merged**